draft-ietf-quic-recovery.md

```diff
@@ -506,9 +506,10 @@ delay sending an acknowledgement.
 The PTO value MUST be set to at least kGranularity, to avoid the timer expiring
 immediately.
 
-When a PTO timer expires, the PTO period MUST be set to twice its current value.
-This exponential reduction in the sender's rate is important because the PTOs
-might be caused by loss of packets or acknowledgements due to severe congestion.
+When a PTO timer expires, the sender sends a probe packet as described in the
```

Comment:
It could be two probe packets, so maybe "sends one or two probe packets"?

Reply:
Or simply "probes the network"?

Reply:
Or simply nothing.  A probe packet is sent if two packets are sent, and this is not normative text.

Reply:
I like @nibanks's suggestion. It's independent of how the probing is actually done, which we describe in detail later.